contributing/3-STYLE-GUIDE.md

```diff
@@ -175,34 +135,8 @@ Here is the live link:
 Note that the live link does not resolve within the GitHub view of this content.
 This link only works on the webpage published to **docs.microsoft.com**.
 
-## Markdown extensions supported by Open Publishing
-The following sections describe supported extensions in Open Publishing.
-
-### Note, warning, tip, important
-Use specific syntax inside a block quote to indicate that the content is a type of note.
-
-```Markdown
-> [!NOTE]
-> This is a note.
-
-> [!WARNING]
-> This is a warning.
-
-> [!TIP]
-> This is a tip.
-
-> [!IMPORTANT]
-> This is important.
-
-```
-
-And it will be rendered like this:
-
-![alert boxes](./images/alert-boxes.png)
-
-## Next steps
-
-See [Formatting code blocks](FORMATTING-CODE.md).
+## Next Steps
+See [Markdown Specifics](4-MARKDOWN-SPECIFICS.md)
```

Comment:
Put a blank line between header and content.  Also add a period at the end of the line with the link.

Reply:
fixed